### ISSUE TITLE
[Screen Reader] Screen reader does not announce search results information by entering 'valid/invalid' data under 'Name' dialog.

### ISSUE THREAD

**Issue by v-kumarsatya** [IgniteUI/igniteui-angular#13524]
Screen reader users rely on auditory feedback to understand the outcome of their actions. If search results are not announced, users won't know whether their search query was successful or if there were any relevant matches.

Screen reader does not announce search results information by entering 'valid/invalid' data under 'Name' dialog.
NVDA remain silent.

Expected Result:

Screen reader should announce search results information by entering 'valid/invalid' data under 'Name' dialog.

source link: [Excel Style Filtering in Angular Data Grid- Ignite UI for Angular (infragistics.com)](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering)

**Comment by v-kumarsatya** [comment]
Request you to provide update on above bug.

**Comment by ChronosSF** [comment]
We are scheduling it for R&D for the next sprint starting Oct 16.

**Comment by v-kumarsatya** [comment]
Any update on this bug.

**Comment by ChronosSF** [comment]
There is a proposed fix but it didn't pass initial reviews so we are still discussing how best to achieve this functionality . We'll continue working on this item next sprint. The earlies you could expect a fix is on November 6

**Comment by v-kumarsatya** [comment]
Is this bug resolved in this sprint or will it take time?
If it will take time, then could you please give an ETA.

**Comment by ChronosSF** [comment]
@v-kumarsatya , we just released new patches that also include the fix for this issue.

**Comment by v-kumarsatya** [comment]
Request you to share Infragistics version number where you have resolved this bug and share the link of demo grid, where we can check above bug.

**Comment by ChronosSF** [comment]
You can check the new functionality with this sample: https://stackblitz.com/edit/4gws68 . The functionality is available in 15.1.34 and 16.1.6
